README.md

```diff
@@ -57,6 +74,73 @@ cargo tauri dev
 ```
 to start a dev server. Alternatively, do `cargo tauri build` to build an installer for your platform. If you want to modify/distribute the app, keep in mind the [license](https://choosealicense.com/licenses/gpl-3.0/#).
 
+### On Linux
+
+***Install the required tools and build chain***
```

Comment:
Great instructions, but I don't know how I feel about these titles. Maybe just have them bold or do "####"?

Reply:
I made them bold instead of italics. Hehe, I just copied what you already had in the existing instructions, that's why it was italics.